Add note regarding Logi Circle API key applications (#13684)
home-assistant/home-assistant.github.io: source/_integrations/logi_circle.markdown

## Requesting API access

> <div class='note warning'>
> 
> Due to the ongoing situation with COVID-19, Logitech have temporarily stopped processing applications for new API accounts. Existing account should continue to work.
> 
> </div>

Reviewer: Small typo and the additional lines are not needed since there is no language context switching happening inside the message.

```suggestion
<div class='note warning'>
Due to the ongoing situation with COVID-19, Logitech has temporarily stopped processing applications for new API accounts. Existing accounts should continue to work.
</div>
```

Author: Yeah I just copied that div from darksky as it was the first place I could think of that had one of those warnings
